Makefile.am

```diff
@@ -135,6 +146,10 @@ libvmi_libvmi_la_CFLAGS= -fvisibility=hidden $(GLIB_CFLAGS) $(JSONC_CFLAGS)
 libvmi_libvmi_la_LDFLAGS= $(GLIB_LIBS) $(JSONC_LIBS) -no-undefined
 libvmi_libvmi_la_LDFLAGS+= -version-info $(VERSION)
 
+if !WITH_KVM_LEGACY
```

Comment:
This whole block is uneccesary, the `AC_CHECK_LIB(kvmi, kvmi_init_vsock` in the configure script will automatically add the required libraries to the linker invocation, you don't have to manually specify. It's also why Jenkins failed.

Reply:
With the wrapper added for kvmi the `AC_CHECK_LIB` call would also need to go btw since no hard linking would be done to kvmi anymore.